src/types/docs-schemas/definitions.schema.json

```diff
@@ -155,6 +155,19 @@
       },
       "required": ["type", "name", "paused", "config"]
     },
+    "category": {
+      "title": "Category",
+      "type": "object",
+      "properties": {
+        "name": { "type": "string" },
+        "description": { "type": "string" },
+        "notificationIds": {
+          "type": "array",
+          "items": { "$ref": "definitions.schema.json/#/definitions/notification", "format": "uri-reference" }
+        }
+      },
+      "required": ["name"]
```

Comment:
can a category be initially created without `notificationIds`?

Reply:
Good catch! Yes, that's intentional. The default categories are also created without any notificationIds.